flang/test/Integration/fp-convert.f90

```diff
@@ -0,0 +1,236 @@
+! RUN: %flang -funsigned %s -o %t && %t | FileCheck %s
+! RUN: %flang -funsigned -emit-llvm -S -o - %s | FileCheck %s --check-prefix=LLVMIR
+
+module fp_convert_m
+  implicit none
+  interface set_and_print
+    module procedure set_and_print_r16
+    module procedure set_and_print_r8
+  end interface
+contains
+  subroutine set_and_print_r16(value)
+    real(kind=16), intent(in) :: value
```

Comment:
real 16 is not enabled on all targets, even for X86-64, it is not always enabled (hence the buildbot failure), you can workaround this by using merge + target dependent check as it was done in https://github.com/llvm/llvm-project/commit/1128343727ce17f65976ef791783cd737f8cb5bd